index.html

```diff
@@ -7,32 +8,34 @@
 </head>
 <body>
 
-<pre id="paste">
-                         _
-     ___ ___ _____ _____|_|___  <span class="sickle">☭</span>
-    |  _| . |     |     | | -_|
-    |___|___|_|_|_|_|_|_|_|___|
-
-    commie is a pastebin script with line commenting support.
-    available at <a href="https://github.com/splitbrain/commie">github</a>. made by <a href="http://www.splitbrain.org">splitbrain</a>.
-
+<div id="content">
 
+<pre>
+  _____                 _
+ |     |___ _____ _____|_|___  <span class="sickle">☭</span>
+ |   --| . |     |     | | -_|
+ |_____|___|_|_|_|_|_|_|_|___|
+</pre>
 
+<div id="paste">
```

Comment:
<b>pre</b> has issues adjusting to mobile. But the ASCII art doesn't really work outside of <b>pre</b>. so that was left in <b>pre</b>, and it's not big enough to be an issue on most phones and mobile devices.

The paste is now pulled into <b>id="paste"</b>  using similar coding defaults to pre, but adjusts to screen size. 

Everything is wrapped in <b>id="content"</b> also to adjust for mobile. it controls the size of content as a whole.